### ISSUE TITLE
docker load does not print the image ID or name

### ISSUE THREAD

**Issue by lucabrunox** [moby/moby#18117]
`docker version`:
Client:
 Version:      1.9.0
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   76d6bc9a
 Built:
 OS/Arch:      linux/amd64

Server:
 Version:      1.9.0
 API version:  1.21
 Go version:   go1.4.3
 Git commit:   76d6bc9a
 Built:
 OS/Arch:      linux/amd64

`docker info`:
Containers: 0
Images: 0
Server Version: 1.9.0
Storage Driver: devicemapper
 Pool Name: docker-254:0-22-pool
 Pool Blocksize: 65.54 kB
 Base Device Size: 1.049 GB
 Backing Filesystem: extfs
 Data file: /dev/loop0
 Metadata file: /dev/loop1
 Data Space Used: 33.88 MB
 Data Space Total: 107.4 GB
 Data Space Available: 1.993 GB
 Metadata Space Used: 466.9 kB
 Metadata Space Total: 52.43 MB
 Metadata Space Available: 51.96 MB
 Udev Sync Supported: false
 Deferred Removal Enabled: false
 Deferred Deletion Enabled: false
 Deferred Deleted Device Count: 0
 Data loop file: /var/lib/docker/devicemapper/devicemapper/data
 Metadata loop file: /var/lib/docker/devicemapper/devicemapper/metadata
 Library Version: 1.02.109 (2015-09-22)
Execution Driver: native-0.2
Logging Driver: json-file
Kernel Version: 4.1.13
Operating System: <unknown>
CPUs: 1
Total Memory: 494.8 MiB
Name: (none)
ID: VEQ6:ODSM:HMVB:3L2U:AVGS:DEDS:GNSU:T2KA:WGJG:FRWH:3PVT:2C72
WARNING: No memory limit support
WARNING: No swap limit support

`uname -a`:
Linux (none) 4.1.13 #1-NixOS SMP Thu Jan 1 00:00:01 UTC 1970 x86_64 GNU/Linux

Steps to reproduce the issue:

```
$ docker load -i image.tar.gz
$
```

Result: the image is loaded correctly, but `docker load` doesn't print anything to stdout.

Expected: I expected the image ID to be printed to stdout

**Comment by MHBauer** [comment]
Load doesn't only do single images. It does entire repositories, which possibly consist of multiple images and tags.

**Comment by lethalman** [comment]
Ok so, isn't it possible to list all of the image ids ? One does not simply know which ones have been loaded without doing a diff with a previous `docker images`.

**Comment by sanmai-NL** [comment]
What's the status of this issue?
Output is being written to stdout instead of stderr, yet up to Docker Engine 1.12.2 it is useless for machines.
If multiple images were imported, print them with a record separator such as newline.

**Comment by thaJeztah** [comment]
Looks like `docker load` now shows the names of images that were loaded;



```bash
docker save -o two-images.tar docker:18.09-dind docker:17.12-dind
# no output

docker load -i two-images.tar
Loaded image: docker:18.09-dind
Loaded image: docker:17.12-dind
```

**Comment by sanmai-NL** [comment]
@thaJeztah That's not a complete resolution. The image ID is less ambiguous when image registry operations happen in parallel. Other tools, such as Podman, do print the image ID and this helps with automation. Please reopen.
